lib/uring/include/discover.ml

```diff
@@ -2,49 +2,122 @@ module C = Configurator.V1
 
 let () =
   C.main ~name:"discover" (fun c ->
-    C.C_define.import c ~c_flags:["-D_GNU_SOURCE"]
-      ~includes:["fcntl.h"; "poll.h"; "sys/uio.h"; "linux/time_types.h"]
-      C.C_define.Type.[
-          "POLLIN", Int;
-          "POLLOUT", Int;
-          "POLLERR", Int;
-          "POLLHUP", Int;
+      let defs =
+        C.C_define.import c ~c_flags:["-D_GNU_SOURCE"; "-I"; Filename.concat (Sys.getcwd ()) "include"]
+          ~includes:["fcntl.h"; "poll.h"; "sys/uio.h"; "linux/time_types.h"; "liburing.h"]
```

Comment:
We can remove "linux/time_types.h" from here if we can successfully include "liburing.h". liburing defines `__kernel_timespce` if not defined already - https://github.com/axboe/liburing/blob/master/configure#L446-L453.

Reply:
OK, I've removed it now.